raiden/storage/sqlite.py

```diff
@@ -8,7 +8,7 @@
 from raiden.utils import typing
 
 # The latest DB version
-RAIDEN_DB_VERSION = 11
+RAIDEN_DB_VERSION = 12
```

Comment:
After my PR was merged this needs to be incremented again

Reply:
done